packages/stream/toit/src/ToitConsumer.js

```diff
@@ -57,14 +57,16 @@ module.exports = class ToitConsumer extends sdk.ConsumerMixin(ToitConnector) {
 					})
 				}
 			} finally {
-				let ackRequest = new toitSubscribeModel.AcknowledgeRequest();
-				ackRequest.setSubscription(subscription);
-				ackRequest.setEnvelopeIdsList(toAcknowledge);
-				this.client.acknowledge(ackRequest, (err) => {
-					if (err) {
-						this.handleWarning(err);
-					}
-				});
+				if (toAcknowledge.length != 0) {
```

Comment:
Just a small comment: can we instead use the strict inequality operator `!==` here? 🙂

Reply:
Grr.
Not coding JavaScript enough :)
done.